mne/channels/tests/test_channels.py

```diff
@@ -236,6 +236,16 @@ def test_get_set_sensor_positions():
     assert_array_equal(raw1.info['chs'][13]['loc'],
                        raw2.info['chs'][13]['loc'])
 
+    for input in ("a_string", 100, raw1, [1, 2]):
+        assert_raises(TypeError, create_1020_montage_mapping,
+                      input)
+    ch_names = ["Cz", "C3", "C4", "C1", "Missing"]
+    intended_output = {"Midline": [0, 3], "Left": [1], "Right": [2]}
+    assert_equal(create_1020_montage_mapping(ch_names),
+                 intended_output)
+    create_1020_montage_mapping(raw1.info)  # test if layout sorting works
```

Comment:
What happens if you pass an info object with a non-10/20 montage (e.g. some BioSemi variant)? I guess it will return a dict where all three keys have empty list values, right? I think it would be good to add this to the test.

Reply:
That's what's happening here :( It's a non-10/20 layout.

Reply:
So it returns a dict with empty lists as expected? OK then. You could create a new info object with the `biosemi64` montage to test a 10/20 one.

Reply:
... eventually? :)

Reply:
I don't quite get it since I didn't run the code. But if you think everything is tested then OK.